vertx-web-client/src/main/java/io/vertx/ext/web/client/WebClientOptions.java

```diff
@@ -59,6 +51,7 @@
   private boolean userAgentEnabled = DEFAULT_USER_AGENT_ENABLED;
   private String userAgent = DEFAULT_USER_AGENT;
   private boolean followRedirects = DEFAULT_FOLLOW_REDIRECTS;
+  private CacheOptions cacheOptions;
```

Comment:
we don't need that anymore since we specify the cache when creating the WebClient instance

Reply:
Would you like to merge `CacheOptions` and `WebClient` options then? I think that currently it provides a better separation.

Reply:
we actually don't need these on WebClientOptions, as the developer should create a WebClient specifying a CacheManager instance.